test/e2e/tests/editor/venues.spec.ts

```diff
@@ -0,0 +1,28 @@
+/**
+ * WordPress dependencies
+ */
+const { test } = require('@wordpress/e2e-test-utils-playwright');
+
+test.describe('Venues in the Editor', () => {
+	test.beforeEach(async ({ admin }) => {
+		await admin.createNewPost({ postType: 'gatherpress_venue' });
+	});
+
+	test('The admin should be able to create a new Venue.', async ({
+		editor,
+		page,
+	}) => {
+		await page.getByLabel('Add title').isVisible();
+		await page.getByLabel('Add title').fill('Test venue');
+		await page.getByLabel('Add title').press('Tab');
+
+		const venue = await page.$('.gatherpress-venue__name');
```

Comment:
The `$` method is discouraged. Locator can be used instead:

```suggestion
		const venue = page.locator('.gatherpress-venue__name');
```

Reply:
Thanks again; this is not only code-cleanup; this is pure education. Thx!